docs: document `HOMEBREW_INSTALL_FROM_API` usage in installation (#13884)
Homebrew/brew: docs/Installation.md

The default Git remote will be used if the corresponding environment variable is unset.

> ## Skip Tap Cloning

Reviewer: ```suggestion
## Skip Tap Cloning (beta)
```

> You can instruct Homebrew to skip cloning the Homebrew/homebrew-core tap during installation by setting `HOMEBREW_INSTALL_FROM_API` in your shell environment with this script:

Reviewer: ```suggestion
You can instruct Homebrew to skip cloning the Homebrew/homebrew-core tap during installation by setting the beta `HOMEBREW_INSTALL_FROM_API` environment variable with the following:
```
